dias/analyzers/find_jump_analyzer.py

```diff
@@ -0,0 +1,1230 @@
+                    # Divide accumulated autos by accumulated weights
+                    # to complete the average
+                    autonorm *= tools.invert_no_zero(weight)
+
+                    # Create a frequency axis representative of the average
+                    dfreq = np.array([(np.mean(this_freq['centre']),
+                                       np.sum(this_freq['width']))],
+                                     dtype=this_freq.dtype)
+
+                else:
+                    # Since freq_collapse was NOT requested, we will apply
+                    # the jump detection algorithm to each frequency
+                    # individually
+                    data = andata.CorrData.from_acq_h5(files,
+                                                       datasets=self.datasets,
+                                                       freq_sel=freq_sel,
+                                                       prod_sel=auto_sel,
+                                                       apply_gain=False,
+                                                       renormalize=False)
+
+                    # Save the index map
+                    this_freq = data.index_map['freq'].copy()
+                    this_input = data.index_map['input'].copy()
+                    this_time = data.time.copy()
+
+                    tstart = data.index_map['time']['ctime'][0]
+
+                    ntime = this_time.size
+                    ninput = this_input.size
+
+                    # Normalize the autocorrelations for each frequency and
+                    # input by the median value over the full time span.
+                    auto = data.vis[:].real
+                    inv_med_auto = tools.invert_no_zero(
+                                   np.median(auto, axis=-1, keepdims=True))
+
+                    autonorm = auto * inv_med_auto - 1.0
+
+                    dfreq = this_freq
+
+                    # Garbage collect
+                    del data, auto, inv_med_auto
+                    gc.collect()
+
+                self.logger.info("Took %0.1f sec to load autocorrelations." %
+                                 (time.time() - t0,))
+
+                # If requested, ignore jumps that occur during solar transit
+                # or near bright source transits
+                tquiet = np.ones(ntime, dtype=np.bool)
+                if self.ignore_sun:
+                    tquiet &= ~_flag_transit('sun', this_time,
+                                             window=self.transit_window)
+
+                if self.ignore_point_sources:
+                    for ss in ["CYG_A", "CAS_A"]:
+                        tquiet &= ~_flag_transit(ss, this_time,
+                                                 window=self.transit_window)
+
+                # Loop over frequencies
+                for ff, freq in enumerate(dfreq):
+
+                    print_cnt = -1
+                    msg = ("Processing frequency %d (%0.2f MHz, %0.2f MHz)" %
```

Comment:
It wouldn't be clear from looking at the log, that the first number is the centre freq and the second is the width.